Makefile

```
@@ -67,7 +67,11 @@ uninstall:
 		"$(DEST)/bin/limactl" \
 		"$(DEST)/bin/nerdctl.lima" \
 		"$(DEST)/share/lima" "$(DEST)/share/doc/lima"
-	# TODO: remove $(DEST)/bin/nerdctl only when it is a symlink to nerdctl.lima
+	@test -f "$(DEST)/bin/nerdctl.lima" && \
```

This condition is always false because you just deleted the file in the line above. Maybe it should be:

```suggestion
	@test -L "$(DEST)/bin/nerdctl" && \
```

---

Thanks for the comment. I didn't know `-L` option. Fixed.